src/component/ImageUploader.tsx

```
@@ -0,0 +1,136 @@
+    }
+  };
+
+  return (
+    <div className="inline-flex flex-col items-start justify-center gap-2">
+      <label htmlFor="image-uploader">Label</label>
+      <div className="flex flex-col gap-2">
+
+          <div
+            role="button"
+            tabIndex={0}
+            className="border-primary-0 relative mr-3 flex h-32 w-32 items-center justify-center rounded-md border-2 border-dashed cursor-pointer focus:outline-none focus:ring-2 focus:ring-primary-0 focus:ring-offset-2"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)